routes/teams.js

```diff
@@ -25,15 +25,19 @@ var http = require("http"),
 
 
 
-router.get('/', function(res, res) {
-      res.render('teams', {
-      });
+router.get('/', function(req, res) {
+    var isLoggedIn = req.user ? true : false;
+  
+      res.render('teams', {user:req.user,loggedIn : isLoggedIn});
```

Comment:
Didn't know how do handlebars #if statement in teams.hbs without passing a boolean